package.json

```diff
@@ -13,39 +13,43 @@
     "css": "WEBPACK_CHILD=css npm run build",
     "test": "jest",
     "coverage": "jest --coverage",
-    "lock": "npm-shrinkwrap --dev",
     "lint": "eslint client/src && sass-lint client/src",
     "lint-js": "eslint client/src",
     "lint-js-fix": "eslint client/src --fix",
     "lint-sass": "sass-lint client/src"
   },
   "dependencies": {
     "@silverstripe/reactstrap-confirm": "^0.0.4",
-    "reactstrap": "^8.5.1",
     "babel-polyfill": "6.7.4",
-    "moment": "^2.22.2",
+    "babel-runtime": "^6.26.0",
```

Comment:
I think this needs to be under `devDependencies`.

Reply:
Have moved